js/gameapp.js

```diff
@@ -160,6 +189,11 @@ gameSearchApp.controller('gameSearchCtrl', ['$http','$scope', function ($http, $
         !(code >= 214 && code <= 216) &&
         !(code >= 224 && code <= 229) &&
         !(code >= 233 && code <= 237) &&
+        !(code == 58) && // colon
```

Comment:
you could do this whole isAlphaNumeric using something to the following: http://stackoverflow.com/questions/25308702/matching-one-or-more-alphanumeric-characters-with-angularjs

if not I am happy with this